docs/components/PkgLayout.tsx

```diff
@@ -15,6 +15,34 @@ import { GithubLogo } from './GithubLogo';
 import { SiteLayout } from './SiteLayout';
 import { PageAlert } from './designSystemComponents';
 
+/**
+ * Calculates what the import statment would be by providing it package data.
+ *
+ * If only 1 import, it outputs a string like this:
+ * import { ComponentName } from '@ag.ds-next/react/component-name';
+ *
+ * If more than 1 import, it outputs a string like this:
+ * import {
+ *     ComponentOne,
+ *     ComponentTwo,
+ * } from '@ag.ds-next/react/component-name'
+ *
+ * This helps keep the string readable and not require scrolling
+ */
+export function calculateImportString(pkg: Pkg) {
```

Comment:
This would benefit from having unit tests associated with it. I'm not sure where those unit tests should be located though. Maybe this function needs to be moved.

Reply:
I've made the unit tests for this now. There might be a better place to put these things though.